src/extensions/vb/Microsoft.DotNet.UpgradeAssistant.Steps.VisualBasic/ExtensionManifest.json

```diff
@@ -0,0 +1,21 @@
+﻿{
+  "ExtensionName": "VB",
+
+  "ConfigUpdater": {
```

Comment:
Only need the ExtensionName and ExtensionServiceProviders sections I think

Reply:
Yep, shouldn't need the rest.